dev-libs/libplatform/libplatform-2.1.0.1-r1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils
+
+MY_PN="platform"
+MY_PN_PREFIX="p8"
+
+DESCRIPTION="Platform support library used by libCEC and binary add-ons for Kodi"
+HOMEPAGE="https://github.com/Pulse-Eight/platform"
+
+LICENSE="GPL-2+"
+SLOT="0"
+IUSE=""
+
+if [[ ${PV} == *9999 ]] ; then
+	EGIT_REPO_URI="https://github.com/Pulse-Eight/${MY_PN}.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/Pulse-Eight/${MY_PN}/archive/${MY_PN_PREFIX}-${MY_PN}-${PV}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/${MY_PN}-${MY_PN_PREFIX}-${MY_PN}-${PV}"
+	KEYWORDS="~amd64 ~arm ~x86"
+fi
+
+src_configure() {
+	local mycmakeargs=(
+		-DBUILD_SHARED_LIBS=ON
```

Comment:
already set by `cmake-utils.eclass` in `EAPI-7`

Reply:
Thanks, I didn't notice that change in cmake-utils.eclass.